src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/esa/EsaIssueFinalDecisionAboutToSubmitHandlerTest.java

```diff
@@ -168,6 +168,41 @@ public void givenANonIssueFinalDecisionEvent_thenReturnFalse() {
         assertFalse(handler.canHandle(ABOUT_TO_SUBMIT, callback));
     }
 
+    @Test
+    public void givenAnIssueFinalDecisionEventRemoveDraftDecisionNotice() {
+        callback.getCaseDetails().getCaseData().getSscsFinalDecisionCaseData().setWriteFinalDecisionPreviewDocument(documentLink);
```

Comment:
```suggestion
        SscsFinalDecisionCaseData finalDecisionCaseData = callback.getCaseDetails().getCaseData().getSscsFinalDecisionCaseData();
```
Can be reused instead of calling it 3 times, same as other test class.